### DISCUSSION TITLE
Policy for owned resources

### DISCUSSION THREAD

**Discussion by tomanovy** [ory/keto#506] [Q&A] [answered]
Hello,
Im trying to understand policies, but at least one think I'm still not getting:
is there a way to set this example "globally" for owned resources
https://www.ory.sh/keto/docs/engines/acp-ory#subject-condition

or there should be extra policy that subject every individual user, could you please point me to right direction? Thanks

In example I'm trying to set something like, but expect this way to be anti-pattern
```
{
  "description": "One policy to rule them all.",
  "subjects": ["users:*"],                    // match any logged user
  "actions": ["delete", "create", "update"],
  "effect": "allow",
  "resources": ["resources:articles:<.*>"],
  "conditions": {
    "owner": {
      "type": "EqualsSubjectCondition", //EqualsCurrentSubject (which matched the pattern)
      "options": {}
    }
  }
}
```

**Comment by vinckr** [comment, marked as answer]
Hey @tomanovy , 
we are releasing a new version of Ory Keto very soon, and I would advise you to switch over to that version (eventually). 
There have been changes over the last version, we are just in the process of writing documentation, but you can check it out already here: 
https://www.ory.sh/keto/docs/next/
Maybe the [subjects page](https://www.ory.sh/keto/docs/next/concepts/subjects) clears up your issue.

**Reply by vinckr** [reply]
Hey @tomanovy,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Also check out the new version of [Ory Keto](https://www.ory.sh/keto/docs/next/)!

Thanks a ton! 🐝
